crates/router/src/connector/cybersource/transformers.rs

```diff
@@ -278,7 +290,7 @@ pub struct CybersourcePaymentsResponse {
     client_reference_information: Option<ClientReferenceInformation>,
 }
 
-#[derive(Default, Debug, Clone, Deserialize, Eq, PartialEq)]
+#[derive(Default, Debug, Clone, Serialize, Deserialize, Eq, PartialEq)]
```

Comment:
Do we absolutely need `Serialize` to be derived here, or can we remove it?

Reply:
We need this because, here we are reusing this struct in PaymentsRequest.